src/api/app/models/comment.rb

```diff
@@ -75,4 +79,10 @@ def blank_or_destroy
   def destroy
     super
   end
+
+  private
+
+  def check_is_child
+    errors[:type] << "is reserved for storing the inheritance class which was not found" unless type.safe_constantize.try(:superclass) == Comment
```

Comment:
According to [Ruby style guide](https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier) it should be in one line:

> Favor modifier if/unless usage when you have a single-line body